libs/api/domains/auth/src/lib/dto/updateDelegation.input.ts

```diff
@@ -0,0 +1,19 @@
+import { Field, InputType } from '@nestjs/graphql'
+
+@InputType('AuthDelegationScopeInput')
+class Scope {
+  @Field((_) => String)
+  name!: string
+
+  @Field((_) => Date)
+  validTo!: Date
+}
+
+@InputType('UpdateAuthDelegationInput')
```

Comment:
AuthUpdateDelegationInput?

cc @saevarma

Reply:
Mutations should be named verb first. This also applies for our naming conventions regarding module names:
```Mutations should still follow the verb first rule. Then the module name, following the object, or “noun,” if applicable.```

https://docs.devland.is/technical-overview/api-design-guide/graphql-naming-conventions#solution

Reply:
Yeah, that documents how to namespace the mutation operation, but the docs don't explicitly address how to name input and payload object types that are used for mutations.

I expected the namespace to come first, but don't mind either way. Would be good to tweak the docs to make this clear.

Reply:
Ahh, didn't even think about that 🙈 .. yeah, the docs need to be tweaked.
IMO `input` names should follow the name of their consumer + `Input`. But yeah, let's take it up in some other thread.